### PR TITLE
 Add xref shortcut to Strings Widget

### PR DESCRIPTION
**Pull request by fcasal**

### CODE REVIEW

**Comment by kernelm0de** [comment]
I think this shortcut will interfere with string widget, because the string widget uses keys to focus on strings starting with that character.

**Comment by ITAYC0HEN** [comment]
indeed, it may cause a problem. 
this issue has been discussed here:
https://github.com/radareorg/cutter/issues/801#issuecomment-429285483

and we couldn't think of a proper shortcut. CTRL+X is an option but it is usually being used for "Cut" and Shift+X is nice

**Comment by Maijin** [comment]
shift nice is usually used for xref from though

**Comment by fcasal** [comment]
> I think this shortcut will interfere with string widget, because the string widget uses keys to focus on strings starting with that character.

There are already other shortcuts that interfere with this such as 's' and 'g' to focus on the search bar. I'd prefer to use X  and maintain a similar shortcut map to IDA.

**Comment by xarkes** [comment]
So?

**Comment by xarkes** [comment]
Hello?

File: src/widgets/StringsWidget.cpp [hunk 1]
```diff
@@ -253,9 +258,9 @@ void StringsWidget::on_actionCopy()
 
     QModelIndex index;
 
-    if(sender() == ui->actionCopy_String)
+    if (sender() == ui->actionCopy_String)
```
Comment (a1ext) [inline src/widgets/StringsWidget.cpp]: I recommend to do `qobject_cast<QAction*>(sender)`  once and then compare it with other actions here

**Comment by ITAYC0HEN** [comment]
ping @fcasal, as Xarkes said

> So as discussed, just add CTRL + X as the shortcut.

**Comment by xarkes** [comment]
Thank you :)

**Pull request merged**